packages/tooling/fast-tooling-wasm/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@microsoft/fast-tooling-wasm",
+  "private": true,
+  "version": "1.0.0",
+  "description": "A set of tooling utilities which has a WASM compile target",
```

Comment:
nit: I do think it's useful to organize this file for readability and consistency similar to how we have other packages.  For example, description after name, licensing / author up near the top as well.  These are just things that first-timers seek out, sometimes even before the readme.